docs/content/reference/resource.md

```diff
@@ -325,6 +325,139 @@ Example:
 mutex: 'alloydb/instance/{{name}}'
 ```
 
+## Sweeper customization
+
+Sweepers are a testing infrastructure mechanism that automatically clean up resources created during tests. They run before tests start and can be run manually to clean up dangling resources. Sweepers help prevent test failures due to resource quota limits and reduce cloud infrastructure costs by removing test resources that were not properly cleaned up. See [sweeper.go ↗](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/resource/sweeper.go).
+
+Sweeper generation is enabled by default, except in the following conditions which require customization here:
+- Resources with custom deletion code
+- Resources with parent-child relationships (unless the parent relationship is configured)
+- Resources with complex URL parameters that aren't simple region/project parameters
+
+Simply defining the sweeper block in a resource will override these exclusions and enable sweeper generation for that resource, even with minimal configuration.
+
+### `exclude_sweeper`
+
+If set to `true`, no sweeper will be generated for this resource. This is useful for resources that cannot or should not be automatically cleaned up.
+
+Default: `false`
+
+Example:
+
+```yaml
+exclude_sweeper: true
+```
+
+### `sweeper`
+
+Configures how test resources are swept (cleaned up) after tests. The sweeper system helps ensure resources created during tests are properly removed, even when tests fail unexpectedly. All fields within the `sweeper` block are optional, with reasonable defaults provided when not specified.
+
+- `identifier_field`: Specifies which field in the resource object should be used to identify resources for deletion. If not specified, defaults to "name" if present in the resource, otherwise falls back to "id".
```

Comment:
```suggestion
- `identifier_field`: Specifies which field in the API resource object should be used to identify resources for deletion. If not specified, defaults to "name" if present in the resource, otherwise falls back to "id".
```